### ISSUE TITLE
[Settings]: Screen Reader is only announcing tab for 'General/Appearance/Advance' tab inside Profile page.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11175]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

Test Environment:
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Open Narrator using 'Win + Ctrl + Enter' keys
Inside PowerShell Profiles page, verify that Screen Reader is only announcing tab for 'General/Appearance/Advance' tab or not

**User Experience:**
Keyboard users are impacted here as they will be unable to know about those control which is confusing for them as result not able to use all functionality properly.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader is only announcing tab for 'GeneralAppearanceAdvance' tab inside Profile page..zip](https://github.com/microsoft/terminal/files/7128377/Screen.Reader.is.only.announcing.tab.for.GeneralAppearanceAdvance.tab.inside.Profile.page.zip)

### Expected Behavior

Screen Reader should announce proper name for 'General/Appearance/Advance' tab inside Profile page. i.e. When 'General' tab is in selected sate and Screen Reader user navigate to it, Screen Reader should announce 'General tab item 1 of 3 Selected'.

### Actual Behavior

Screen Reader is only announcing 'tab' for 'General/Appearance/Advance' tab inside Profile page. i.e. When 'General' tab is in selected sate and Screen Reader user navigate to it, Screen Reader is announcing 'tab'. 

**Observation:** Issue not repro when user navigate with left/right arrow key inside that tab section. 
Same issue repro with NVDA Screen Reader.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as UI has been changed.
